gazebo_plugins/src/gazebo_ros_wheel_slip.cpp

```diff
@@ -34,6 +36,17 @@ class GazeboRosWheelSlipPrivate
 
   /// Handle to parameters callback
   rclcpp::Node::OnSetParametersCallbackHandle::SharedPtr on_set_parameters_callback_handle_;
+
+  // Containers to hold default values of slip parameters
+  std::map<std::string, double> mapSlipLateralDefault;
+  double default_slip_lateral;
+  std::map<std::string, double> mapSlipLongitudinalDefault;
+  double default_slip_longitudinal;
+
+  // Event handler to set slip compliance values for individual wheels based
+  std::shared_ptr<rclcpp::ParameterEventHandler> parameter_subscriber;
```

Comment:
nitpick: For clarity, I would name the variable to reflect the type, e.g. `parameter_event_handler_`